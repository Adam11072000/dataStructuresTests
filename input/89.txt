init 214
mergeSuperPixels 6346 147 214
setLabelScore 4252 115 4802 4456
addImage -9
addImage 4459
setLabelScore -5 15 2005 8919
deleteImage 4459
addImage 7663
mergeSuperPixels 7663 176 0
addImage -5
addImage 9478
addImage 5905
mergeSuperPixels 5905 143 0
addImage 0
addImage 3274
setLabelScore 6000 -9 6461 2026
mergeSuperPixels 3274 8 0
mergeSuperPixels 3274 200 1
mergeSuperPixels 7663 189 1
setLabelScore 5905 10 4926 4492
resetLabelScore 3274 66 0
mergeSuperPixels 4758 100 -9
setLabelScore 3274 190 4550 3225
setLabelScore 3274 213 1243 6802
addImage 1924
mergeSuperPixels 6416 221 53
mergeSuperPixels 7663 185 2
getHighestScoredLabel 0 218
resetLabelScore 9478 80 0
deleteImage 5905
setLabelScore 1924 26 5249 3271
resetLabelScore 2611 66 -3
setLabelScore 7432 220 9234 874
resetLabelScore 5833 223 5993
mergeSuperPixels 3274 108 2
addImage 1924
setLabelScore 5247 108 5847 -3
setLabelScore 1924 196 524 5397
addImage 4655
setLabelScore 7062 218 7977 2240
setLabelScore 7663 128 9702 2560
resetLabelScore -1 0 2881
setLabelScore 9478 17 3922 3371
setLabelScore 9478 122 7381 3807
mergeSuperPixels 4142 178 -1
setLabelScore 9478 212 5219 3053
mergeSuperPixels 9478 166 0
deleteImage 7663
resetLabelScore 9478 152 0
resetLabelScore 9478 108 0
setLabelScore 6611 -2 8616 4236
setLabelScore 6240 188 -3 8376
setLabelScore 3087 93 -5 9158
addImage -7
resetLabelScore -3 192 4066
mergeSuperPixels 4323 20 218
mergeSuperPixels 9478 123 1
resetLabelScore 242 110 -2
addImage 0
addImage 8939
setLabelScore 3274 52 3414 7069
setLabelScore 9478 92 2435 9574
getHighestScoredLabel 1924 144
addImage -5
getHighestScoredLabel 3274 69
mergeSuperPixels 1924 70 0
mergeSuperPixels 8939 167 0
mergeSuperPixels 9478 151 2
setLabelScore 4655 207 2274 1678
resetLabelScore 2598 -5 6546
setLabelScore 4655 67 7855 8134
getHighestScoredLabel 0 214
setLabelScore 1924 154 9962 5388
getHighestScoredLabel 4655 139
addImage 1924
deleteImage 1924
setLabelScore 2148 50 8647 8554
addImage 4655
resetLabelScore 6397 70 -4
addImage 8670
setLabelScore 9478 137 1581 5441
mergeSuperPixels 4655 82 0
mergeSuperPixels 6940 -6 139
resetLabelScore 6556 86 174
setLabelScore 8939 93 3396 9082
resetLabelScore 4360 71 -5
getHighestScoredLabel 3274 26
mergeSuperPixels 4655 155 1
resetLabelScore 4655 38 0
mergeSuperPixels 9478 7 3
addImage -6
addImage 8939
getHighestScoredLabel 3274 157
getHighestScoredLabel 4655 211
setLabelScore 8939 213 2495 4034
getHighestScoredLabel 8670 84
resetLabelScore 6539 -6 5201
resetLabelScore 8565 -4 4900
setLabelScore 6051 218 4326 4508
setLabelScore 3744 41 -7 7518
addImage 3192
resetLabelScore 3274 184 0
mergeSuperPixels 9478 195 4
setLabelScore 8696 -5 6409 8910
resetLabelScore 5880 30 6936
mergeSuperPixels 1192 -1 201
setLabelScore 4655 32 2337 9072
addImage 3377
resetLabelScore 8670 84 0
mergeSuperPixels 8670 201 0
resetLabelScore 3438 22 -2
mergeSuperPixels 3377 138 0
setLabelScore 9478 77 192 2605
resetLabelScore 8670 195 0
mergeSuperPixels 8939 150 1
mergeSuperPixels 8670 173 72
getHighestScoredLabel 3377 99
mergeSuperPixels 1 -5 87
mergeSuperPixels 3274 134 55
resetLabelScore 0 196 6012
mergeSuperPixels 3192 0 1
mergeSuperPixels 8939 66 2
getHighestScoredLabel 9478 145
getHighestScoredLabel 3377 88
addImage -7
mergeSuperPixels 4655 154 2
addImage -1
addImage -3
addImage 0
setLabelScore 8009 61 -5 2228
resetLabelScore -5 20 918
setLabelScore 1948 161 0 4493
mergeSuperPixels 3377 21 99
setLabelScore 8670 204 8928 4068
mergeSuperPixels 2858 153 -2
resetLabelScore 8939 182 0
setLabelScore 9404 13 -9 6713
getHighestScoredLabel 8939 181
setLabelScore 4655 41 9052 2036
mergeSuperPixels 3274 113 3
setLabelScore 9478 128 338 4395
setLabelScore 9478 19 7966 5233
setLabelScore 3377 67 3841 6878
mergeSuperPixels 3274 112 4
resetLabelScore 1032 216 5291
getHighestScoredLabel 8939 131
setLabelScore 9870 161 -7 320
getHighestScoredLabel 4655 78
addImage 7638
setLabelScore 8939 112 4650 5731
addImage 7260
resetLabelScore 8786 133 6447
resetLabelScore 9058 91 287
addImage -1
resetLabelScore 8939 41 0
addImage 7260
getHighestScoredLabel 9478 1
mergeSuperPixels 8796 219 159
mergeSuperPixels 7260 22 0
getHighestScoredLabel 8939 94
addImage 4939
resetLabelScore 7591 20 -1
addImage 8939
addImage 6917
resetLabelScore 7361 35 -6
addImage 8396
mergeSuperPixels 3274 164 5
setLabelScore 9478 117 292 9877
getHighestScoredLabel 9478 68
resetLabelScore 3274 192 0
setLabelScore 5341 133 5910 -2
getHighestScoredLabel 3274 179
addImage -7
addImage 5809
resetLabelScore 9000 -10 1857
addImage 4040
getHighestScoredLabel 8396 15
setLabelScore 8396 84 3600 3034
setLabelScore 2905 203 9035 7351
setLabelScore 9478 153 4621 6518
resetLabelScore 3377 91 0
resetLabelScore -5 148 7659
mergeSuperPixels 3192 143 1
mergeSuperPixels 3192 152 2
mergeSuperPixels 3127 81 84
setLabelScore -8 160 6473 4566
setLabelScore 3377 86 330 7457
mergeSuperPixels 7905 -3 13
setLabelScore 5999 12 4891 239
deleteImage 9566
addImage 6917
setLabelScore 6750 56 -5 7392
mergeSuperPixels 4130 -4 4
mergeSuperPixels 7050 -2 99
resetLabelScore 6022 218 9857
resetLabelScore 4627 60 -7
setLabelScore 4655 90 767 1961
setLabelScore 7638 112 5037 1590
setLabelScore 8670 41 9622 248
resetLabelScore 1352 41 3666
addImage 1443
mergeSuperPixels 4367 98 -10
mergeSuperPixels 7638 62 129
setLabelScore 9478 200 2021 7447
mergeSuperPixels 3377 61 1
getHighestScoredLabel 7260 74
getHighestScoredLabel 8396 136
addImage 6888
setLabelScore 3377 10 7066 6152
resetLabelScore 8670 172 0
getHighestScoredLabel 3192 36
setLabelScore 8396 29 4809 8376
setLabelScore 8670 169 3357 7070
setLabelScore 5809 112 7170 3722
setLabelScore 188 214 3189 2724
deleteImage -3
addImage 1852
mergeSuperPixels 7638 42 0
getHighestScoredLabel 7638 46
setLabelScore 3377 41 97 1503
mergeSuperPixels 6032 2 -7
resetLabelScore 1801 87 0
setLabelScore 4150 101 -8 9978
getHighestScoredLabel 1443 79
getHighestScoredLabel 4939 51
mergeSuperPixels 1852 89 0
addImage 7638
mergeSuperPixels 9367 -3 58
setLabelScore 6888 177 8771 769
mergeSuperPixels 9660 173 97
deleteImage 2779
resetLabelScore -7 82 8961
addImage 3424
mergeSuperPixels 5809 146 0
mergeSuperPixels 4880 119 215
mergeSuperPixels 9478 173 5
mergeSuperPixels 5023 87 214
getHighestScoredLabel 3192 33
setLabelScore 6917 95 7470 2115
resetLabelScore 3377 140 0
setLabelScore 3274 168 208 9602
setLabelScore 4655 105 4768 5460
mergeSuperPixels 2831 -9 168
setLabelScore 9478 131 6367 6652
addImage 5190
setLabelScore 6285 180 6678 3193
getHighestScoredLabel 4655 74
mergeSuperPixels 6888 16 0
resetLabelScore 7638 123 0
addImage 9237
addImage -2
mergeSuperPixels 4655 94 3
setLabelScore 8104 97 -3 4243
setLabelScore 7770 174 -3 9587
addImage 1587
mergeSuperPixels 4571 48 -5
addImage 6285
addImage 7360
getHighestScoredLabel 3424 16
mergeSuperPixels -6 45 151
mergeSuperPixels 2671 174 214
resetLabelScore 9844 215 4538
addImage -4
deleteImage -10
getHighestScoredLabel 3377 5
mergeSuperPixels 8396 34 169
setLabelScore 4839 0 -10 4802
getHighestScoredLabel 0 223
addImage 8165
addImage 532
getHighestScoredLabel 5190 102
addImage 306
mergeSuperPixels 7314 101 208
deleteImage 808
resetLabelScore 1687 170 2540
addImage 6068
setLabelScore 1852 80 8356 9636
setLabelScore 5190 178 5696 5041
resetLabelScore 1386 -2 2260
mergeSuperPixels 1587 187 0
setLabelScore 5190 27 9035 7930
mergeSuperPixels 306 190 0
setLabelScore 4957 0 738 -8
resetLabelScore 9478 14 0
addImage 7074
mergeSuperPixels 532 73 0
addImage 1321
resetLabelScore 4040 207 0
addImage 8641
addImage -7
mergeSuperPixels 532 57 1
resetLabelScore 3192 165 0
setLabelScore 8670 66 1991 6196
mergeSuperPixels 2766 142 221
setLabelScore 543 208 -6 3762
mergeSuperPixels 1443 50 0
resetLabelScore -2 29 6589
getHighestScoredLabel 532 199
setLabelScore 3274 145 2430 4689
mergeSuperPixels 3274 0 6
setLabelScore 9237 125 950 3978
getHighestScoredLabel 8939 82
setLabelScore 9478 183 3296 9899
setLabelScore 8165 146 1834 4948
setLabelScore 6285 29 745 6354
mergeSuperPixels 7260 7 1
mergeSuperPixels 8670 190 1
resetLabelScore 0 45 3444
resetLabelScore 764 197 2124
addImage 3548
addImage -9
resetLabelScore 8322 -9 7820
deleteImage -10
setLabelScore 7260 72 8660 5128
setLabelScore 5943 -6 9491 5891
setLabelScore 7360 28 9078 7819
resetLabelScore 7459 222 701
setLabelScore 5809 115 9373 9383
resetLabelScore 0 133 7010
mergeSuperPixels 6285 114 0
setLabelScore 6068 189 3575 3177
addImage 1151
mergeSuperPixels 6888 203 1
setLabelScore 3228 36 2500 8780
addImage 9431
addImage 4035
addImage 5908
mergeSuperPixels 3274 155 7
addImage 438
mergeSuperPixels 6888 145 2
resetLabelScore 230 76 -9
setLabelScore 8670 200 2541 550
mergeSuperPixels 3377 208 2
getHighestScoredLabel 8641 169
mergeSuperPixels 7360 123 0
getHighestScoredLabel 5908 16
mergeSuperPixels -6 142 0
deleteImage 5046
setLabelScore 1852 32 1111 76
setLabelScore -1 49 2249 8679
resetLabelScore 9645 209 -9
resetLabelScore 9964 114 5329
mergeSuperPixels 9308 214 203
setLabelScore 4058 33 1869 -3
getHighestScoredLabel 6068 112
addImage 8939
resetLabelScore 5496 -8 4080
addImage -4
resetLabelScore 6529 29 5749
resetLabelScore 6917 136 0
addImage 2991
mergeSuperPixels 1587 205 1
setLabelScore 4052 131 480 7171
resetLabelScore -5 49 2370
mergeSuperPixels 6917 121 0
setLabelScore 6399 223 7207 7760
setLabelScore 3274 131 9507 2803
setLabelScore 330 117 -2 7329
mergeSuperPixels 9478 10 6
mergeSuperPixels 3548 197 0
getHighestScoredLabel 7074 66
getHighestScoredLabel 9478 127
getHighestScoredLabel 6888 198
setLabelScore 5818 154 -4 2270
addImage -10
setLabelScore 6068 195 770 5479
mergeSuperPixels 4035 74 156
setLabelScore 8396 76 6210 8668
getHighestScoredLabel 4655 4
deleteImage 4035
mergeSuperPixels 8165 113 0
resetLabelScore 5190 42 0
resetLabelScore 4939 59 0
setLabelScore 9478 21 2877 5468
mergeSuperPixels 8670 22 2
setLabelScore 532 75 2644 6777
setLabelScore 3522 197 8496 -2
addImage 4782
mergeSuperPixels 5809 24 1
mergeSuperPixels 7828 214 72
getHighestScoredLabel 5190 9
resetLabelScore 6336 123 5100
addImage 1522
deleteImage 3479
resetLabelScore 8750 19 -4
setLabelScore 7260 0 1692 2052
mergeSuperPixels 8670 142 3
resetLabelScore 8939 160 0
addImage 1941
getHighestScoredLabel 7074 66
addImage 1981
mergeSuperPixels 6932 -5 36
addImage 886
resetLabelScore 306 4 0
addImage 253
setLabelScore 9431 110 3389 109
resetLabelScore 2991 15 0
deleteImage 7065
mergeSuperPixels 306 116 1
setLabelScore 6846 110 4571 1697
addImage 2559
setLabelScore 4655 154 4959 3479
resetLabelScore 532 195 0
addImage 8171
resetLabelScore 8641 53 0
resetLabelScore 5981 158 149
setLabelScore 8165 0 9992 6160
setLabelScore 6587 71 -6 5970
setLabelScore 7260 14 559 933
mergeSuperPixels 6068 142 0
setLabelScore 9410 179 6188 5585
addImage -4
addImage -3
setLabelScore 3377 66 9075 5299
setLabelScore 7641 58 4841 4631
getHighestScoredLabel 8165 78
setLabelScore 6285 144 1586 9261
getHighestScoredLabel 6917 199
setLabelScore 1443 76 8262 8682
mergeSuperPixels 6888 96 3
mergeSuperPixels 253 5 0
mergeSuperPixels 6338 216 61
mergeSuperPixels 9921 108 -4
mergeSuperPixels 6917 72 1
resetLabelScore 3377 70 0
setLabelScore -4 89 577 5792
addImage 4566
getHighestScoredLabel 8396 34
quit