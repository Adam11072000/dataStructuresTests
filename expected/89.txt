init done.
mergeSuperPixels: INVALID_INPUT
setLabelScore: FAILURE
addImage: INVALID_INPUT
addImage: SUCCESS
setLabelScore: INVALID_INPUT
deleteImage: SUCCESS
addImage: SUCCESS
mergeSuperPixels: SUCCESS
addImage: INVALID_INPUT
addImage: SUCCESS
addImage: SUCCESS
mergeSuperPixels: SUCCESS
addImage: INVALID_INPUT
addImage: SUCCESS
setLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
setLabelScore: SUCCESS
setLabelScore: SUCCESS
addImage: SUCCESS
mergeSuperPixels: INVALID_INPUT
mergeSuperPixels: SUCCESS
getHighestScoredLabel: INVALID_INPUT
resetLabelScore: INVALID_INPUT
deleteImage: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
addImage: FAILURE
setLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
addImage: SUCCESS
setLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: INVALID_INPUT
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
deleteImage: SUCCESS
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
resetLabelScore: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
addImage: SUCCESS
setLabelScore: SUCCESS
setLabelScore: SUCCESS
getHighestScoredLabel: FAILURE
addImage: INVALID_INPUT
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
getHighestScoredLabel: INVALID_INPUT
setLabelScore: SUCCESS
getHighestScoredLabel: FAILURE
addImage: FAILURE
deleteImage: SUCCESS
setLabelScore: FAILURE
addImage: FAILURE
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: INVALID_INPUT
resetLabelScore: FAILURE
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
addImage: INVALID_INPUT
addImage: FAILURE
getHighestScoredLabel: FAILURE
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
getHighestScoredLabel: FAILURE
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
setLabelScore: INVALID_INPUT
resetLabelScore: FAILURE
mergeSuperPixels: INVALID_INPUT
setLabelScore: SUCCESS
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
getHighestScoredLabel: FAILURE
mergeSuperPixels: INVALID_INPUT
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
getHighestScoredLabel: FAILURE
getHighestScoredLabel: FAILURE
addImage: INVALID_INPUT
mergeSuperPixels: SUCCESS
addImage: INVALID_INPUT
addImage: INVALID_INPUT
addImage: INVALID_INPUT
setLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: INVALID_INPUT
resetLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
setLabelScore: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
getHighestScoredLabel: FAILURE
setLabelScore: INVALID_INPUT
getHighestScoredLabel: FAILURE
addImage: SUCCESS
setLabelScore: SUCCESS
addImage: SUCCESS
resetLabelScore: FAILURE
resetLabelScore: FAILURE
addImage: INVALID_INPUT
resetLabelScore: INVALID_INPUT
addImage: FAILURE
getHighestScoredLabel: FAILURE
mergeSuperPixels: INVALID_INPUT
mergeSuperPixels: SUCCESS
getHighestScoredLabel: FAILURE
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: FAILURE
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
getHighestScoredLabel: FAILURE
resetLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
getHighestScoredLabel: FAILURE
addImage: INVALID_INPUT
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
setLabelScore: FAILURE
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: FAILURE
setLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
mergeSuperPixels: INVALID_INPUT
setLabelScore: FAILURE
deleteImage: FAILURE
addImage: FAILURE
setLabelScore: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
setLabelScore: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: FAILURE
addImage: SUCCESS
mergeSuperPixels: INVALID_INPUT
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
getHighestScoredLabel: FAILURE
getHighestScoredLabel: FAILURE
addImage: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
setLabelScore: SUCCESS
setLabelScore: SUCCESS
setLabelScore: INVALID_INPUT
deleteImage: INVALID_INPUT
addImage: SUCCESS
mergeSuperPixels: SUCCESS
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
mergeSuperPixels: INVALID_INPUT
resetLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
getHighestScoredLabel: FAILURE
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
addImage: FAILURE
mergeSuperPixels: INVALID_INPUT
setLabelScore: SUCCESS
mergeSuperPixels: FAILURE
deleteImage: FAILURE
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: INVALID_INPUT
mergeSuperPixels: SUCCESS
mergeSuperPixels: INVALID_INPUT
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: INVALID_INPUT
setLabelScore: SUCCESS
addImage: SUCCESS
setLabelScore: FAILURE
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
addImage: INVALID_INPUT
mergeSuperPixels: SUCCESS
setLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
addImage: SUCCESS
mergeSuperPixels: INVALID_INPUT
addImage: SUCCESS
addImage: SUCCESS
getHighestScoredLabel: FAILURE
mergeSuperPixels: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
resetLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
deleteImage: INVALID_INPUT
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
setLabelScore: INVALID_INPUT
getHighestScoredLabel: INVALID_INPUT
addImage: SUCCESS
addImage: SUCCESS
getHighestScoredLabel: FAILURE
addImage: SUCCESS
mergeSuperPixels: FAILURE
deleteImage: FAILURE
resetLabelScore: FAILURE
addImage: SUCCESS
setLabelScore: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
mergeSuperPixels: SUCCESS
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
addImage: INVALID_INPUT
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
mergeSuperPixels: INVALID_INPUT
setLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
setLabelScore: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
resetLabelScore: FAILURE
addImage: SUCCESS
addImage: INVALID_INPUT
resetLabelScore: INVALID_INPUT
deleteImage: INVALID_INPUT
setLabelScore: SUCCESS
setLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
addImage: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: FAILURE
addImage: SUCCESS
addImage: SUCCESS
addImage: SUCCESS
mergeSuperPixels: SUCCESS
addImage: SUCCESS
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
getHighestScoredLabel: FAILURE
mergeSuperPixels: INVALID_INPUT
deleteImage: FAILURE
setLabelScore: SUCCESS
setLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
resetLabelScore: FAILURE
mergeSuperPixels: INVALID_INPUT
setLabelScore: INVALID_INPUT
getHighestScoredLabel: FAILURE
addImage: FAILURE
resetLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
resetLabelScore: FAILURE
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: FAILURE
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
setLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
setLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
getHighestScoredLabel: FAILURE
getHighestScoredLabel: FAILURE
getHighestScoredLabel: FAILURE
setLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
getHighestScoredLabel: FAILURE
deleteImage: SUCCESS
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
setLabelScore: INVALID_INPUT
addImage: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: INVALID_INPUT
getHighestScoredLabel: FAILURE
resetLabelScore: FAILURE
addImage: SUCCESS
deleteImage: FAILURE
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
getHighestScoredLabel: FAILURE
addImage: SUCCESS
mergeSuperPixels: INVALID_INPUT
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
deleteImage: FAILURE
mergeSuperPixels: SUCCESS
setLabelScore: FAILURE
addImage: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
resetLabelScore: FAILURE
setLabelScore: SUCCESS
setLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: FAILURE
addImage: INVALID_INPUT
addImage: INVALID_INPUT
setLabelScore: SUCCESS
setLabelScore: FAILURE
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
addImage: SUCCESS
getHighestScoredLabel: FAILURE
quit done.
